esrally/log.py

```diff
@@ -95,16 +95,29 @@ def install_default_log_config():
         source_path = io.normalize_path(os.path.join(os.path.dirname(__file__), "resources", "logging.json"))
         with open(log_config, "w", encoding="UTF-8") as target:
             with open(source_path, encoding="UTF-8") as src:
-                # Ensure we have a trailing path separator as after LOG_PATH there will only be the file name
-                log_path = os.path.join(paths.logs(), "")
-                # the logging path might contain backslashes that we need to escape
-                log_path = io.escape_path(log_path)
```

Comment:
We're dropping backlash escaping which was meant to help with Windows judging by https://github.com/elastic/rally/pull/829. But we never officially supported Windows so I'm fine with it.

Reply:
Oh, how did I not see this? I am happy to add this logic back to the code.